frontends/ol-components/src/components/Chips/Chip.stories.tsx

```diff
@@ -0,0 +1,102 @@
+import React from "react"
+import type { Meta, StoryObj } from "@storybook/react"
+import Chip from "@mui/material/Chip"
+import Stack from "@mui/material/Stack"
+import { fn } from "@storybook/test"
+import { ChipLink } from "./ChipLink"
+import { withRouter } from "storybook-addon-react-router-v6"
+import CalendarTodayIcon from "@mui/icons-material/CalendarToday"
+
+const COLORS = ["default", "primary", "secondary"] as const
+const VARIANTS = [
+  {
+    variant: "filled",
+    label: "Filled",
+  },
+  {
+    variant: "outlined",
+    label: "Outlined",
+  },
+] as const
+
+const meta: Meta<typeof Chip> = {
+  title: 'smoot-design/Chip ("Pill")',
+  component: Chip,
+  render: (args) => {
```

Comment:
The stories are missing their controls in the Storybook UI. Perhaps not much value if we're just wrapping the MUI component, but could be useful for designers if we can import/derive them from @mui/material without too much trouble.

Reply:
I added some controls. Apparently storybook infers controls via a tool [react-docgen](https://github.com/reactjs/react-docgen) that scales the files, and doesn't work great with MUI components. I didn't look into it much, but that's why I provided specific argTypes.

Overall, my goal with the Chip and Button stories was mostly making the Docs page where you can see all the variations. Then it's easy to see how a tweak to the code changes the different possible buttons/chips/wahtever.

That said, adding simple controls is easy and definitely improvement 👍 👍